src/bisection.jl

```diff
@@ -46,6 +46,11 @@ function SciMLBase.solve(prob::IntervalNonlinearProblem, alg::Bisection, args...
                     retcode = ReturnCode.FloatingPointLimit,
                     left = left, right = right)
             fm = f(mid)
+            if abs((right - left) / 2) < atol
+                return SciMLBase.build_solution(prob, alg, mid, fm;
+                    retcode = ReturnCode.Success,
```

Comment:
@ChrisRackauckas in case of tolerance reached, we would naturally return the 'Success' retcode, right?

Reply:
yes